src/containers/orders/order/order-table/tests/order-table.test.js

```diff
@@ -1,35 +1,164 @@
 };
-let wrapper;
+
+jest.mock('react-redux');
+jest.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (key) => key,
+    i18n: { changeLanguage: jest.fn() }
+  })
+}));
 
 const props = {
-  currency: 0,
-  calcPrice: 100,
-  cartLoading: false,
+  cartLoading: true,
   cartQuantityLoading: false,
-  items: [{ product: { product_id: 'some id' }, price: 10 }],
+  items: [
+    {
+      product: { product_id: 'some id' },
+      sizeAndPrice: {
+        price: [
+          {
+            value: 1000,
+            currency: 'UAH'
+          },
+          {
+            value: 37,
+            currency: 'USD'
+          }
+        ],
+        size: { _id: 'test', name: 'S' }
+      }
+    }
+  ],
   user: {}
 };
-describe('Order table component tests', () => {
-  it('should match snapshot', () => {
+
+const modalProps = {
+  language: 1,
+  message: 'test',
+  isOpen: true,
+  onAction: jest.fn()
+};
+
+const mockGetCartItem = jest.fn().mockReturnValue({
+  sizeAndPrice: {
+    size: {
+      _id: 'test',
+      name: 'S'
+    },
+    price: [
+      {
+        value: 1000,
+        currency: 'UAH'
+      },
+      {
+        value: 37,
+        currency: 'USD'
+      }
+    ]
+  }
+});
+
+const mockGetTotalPrice = jest.fn();
+const mockChangeQuantity = jest.fn();
+const mockChangeSize = jest.fn();
+
+const mockCartOperations = {
+  getTotalPrice: mockGetTotalPrice,
+  changeQuantity: mockChangeQuantity,
+  getCartItem: mockGetCartItem,
+  changeSize: mockChangeSize,
+  removeFromCart: jest.fn()
+};
+
+describe('test <OrderTable /> component', () => {
+  it('should render <OrderTable />', () => {
     testUseSelector(0);
-    wrapper = shallow(<OrderTable {...props} cartOperations={mockCartOperations} />);
-    expect(wrapper).toMatchSnapshot();
+    const { getByText } = render(
+      <BrowserRouter>
+        <OrderTable {...props} cartOperations={mockCartOperations} />
+      </BrowserRouter>
+    );
+    const productCell = getByText(/product/i);
+    expect(productCell).toBeInTheDocument();
+  });
+});
+
+describe('test <Modal/> component', () => {
+  beforeEach(() => {
+    render(<Modal {...modalProps} />);
+  });
+
+  it('should render <Modal /> component', () => {
+    expect(screen.queryByText(modalProps.message)).toBeInTheDocument();
+    expect(screen.queryByText('common.buttons.confirm')).toBeInTheDocument();
+    expect(screen.queryByText('common.buttons.cancel')).toBeInTheDocument();
+    expect(screen.queryByText('common.modalHeader')).toBeInTheDocument();
+  });
+
+  it('call onAction with true', () => {
+    const confirmButton = screen.queryByText('common.buttons.confirm');
+
+    expect(confirmButton).toBeInTheDocument();
+    act(() => {
+      fireEvent.click(confirmButton);
+    });
+    expect(modalProps.onAction).toHaveBeenCalledWith(true);
+  });
+
+  it('call onAction with false', () => {
+    const cancelButton = screen.queryByText('common.buttons.cancel');
+
+    expect(cancelButton).toBeInTheDocument();
```

Comment:
This expects don't do anything, cause if fireEvent is triggered, then it's obvious that we have fireEvent.target in the document.
Same for all test cases besides first